openshift_cli_installer/utils/gcp_utils.py

```diff
@@ -0,0 +1,60 @@
+import os
+import shutil
+from pathlib import Path
+import tempfile
+
+from simple_logger.logger import get_logger
+from openshift_cli_installer.utils.const import GCP_STR
+
+LOGGER = get_logger(name=__name__)
+
+
+def set_gcp_configuration(user_input):
+    """
+    Saves provided GCP Service Account file at location '~/.gcp/osServiceAccount.json'
+
+    If file already exists at this path, it will be copied to tmp file first.
+
+    Returns:
+        dict or None: A dictionary of parameters needed for setting/restoring
```

Comment:
once you add `gcp_params = {}` this should stay `dict` :)

Reply:
updated